test/helpers/TestUtils.php

```diff
@@ -135,4 +143,11 @@ public static function enableRewritePathPattern($htaccessFilePath, $langIdentifi
             file_put_contents($htaccessFilePath, $replacedHtaccess);
         }
     }
+
+    public static function enableWovnJson($htaccessFilePath)
+    {
+        $htaccess = file_get_contents($htaccessFilePath);
+        $replacedHtaccess = $htaccess . "\nSetEnv WOVN_CONFIG";
+        file_put_contents($htaccessFilePath, $replacedHtaccess);
+    }
```

Comment:
SetEnv in .htaccess means that set variable into `$_SERVER['WOVN_CONFIG']`;
So, this method should be like the following.
```
public static function enableWovnJson($htaccessFilePath, $configFilePath)
    {
        $htaccess = file_get_contents($htaccessFilePath);
        $replacedHtaccess = $htaccess . "\nSetEnv WOVN_CONFIG " . $configFilePath;
        error_log($replacedHtaccess);
        file_put_contents($htaccessFilePath, $replacedHtaccess);
    }
```

Reply:
Yeah, I see what you mean. For this one I took a shortcut by simply setting the env variable (leaving it blank). And when `_SERVER['WOVN_CONFIG']` is set but is blank, WOVN.php will try to load `wovn.ini` from the default location of `{document_root}/wovn.ini`. It's a bit hacky. 😅 Will update it to the proper way.